internal/provider/resource_vpc_firewall_rules.go

```diff
@@ -676,7 +676,9 @@ func newFilterTypeFromModel(model *vpcFirewallRulesResourceRuleFiltersModel) oxi
 	protocols := []oxide.VpcFirewallRuleProtocol{}
 	for _, protocol := range model.Protocols.Elements() {
 		p, _ := strconv.Unquote(protocol.String())
-		protocols = append(protocols, oxide.VpcFirewallRuleProtocol(p))
+		protocols = append(protocols, oxide.VpcFirewallRuleProtocol{
+			Type: oxide.VpcFirewallRuleProtocolType(p),
+		})
```

Comment:
This pull request is not finished yet. I need to update the schema to account for the fact that `oxide.VpcFirewallRuleProtocol` now has `Type` and `Value` fields.

Reply:
Addressed in https://github.com/oxidecomputer/terraform-provider-oxide/pull/474/commits/c6c7ca511279ec856f6f3cafa641d9cf9ee006f1.